pipeline-braille-utils/liblouis/liblouis-tables/pom.xml

```diff
@@ -12,15 +12,15 @@
 
   <groupId>org.daisy.pipeline.modules.braille</groupId>
   <artifactId>liblouis-tables</artifactId>
-  <version>2.5.4-p1-SNAPSHOT</version>
+  <version>2.6.2-SNAPSHOT</version>
   <packaging>bundle</packaging>
 
   <name>DAISY Pipeline 2 Braille Module :: liblouis-tables</name>
 
   <properties>
-    <liblouis.version>2.5.4</liblouis.version>
-    <liblouis.tarball.url>http://www.liblouis.org/downloads/liblouis-2.5.4.tar.gz</liblouis.tarball.url>
-    <liblouis.tarball.sha1>d2c6727973760560235daab7a8830f19b31adb49</liblouis.tarball.sha1>
+    <liblouis.version>2.6.2</liblouis.version>
+    <liblouis.tarball.url>https://github.com/liblouis/liblouis/releases/download/v2.6.2/liblouis-2.6.2.tar.gz</liblouis.tarball.url>
```

Comment:
This build doesn't include the new Finnish tables yet. @egli have you uploaded the right v2.6.2 (i.e. after the last minute change)?